packages/babel-generator/src/generators/jsx.js

```diff
@@ -92,3 +92,26 @@ export function JSXClosingElement(node: Object) {
 }
 
 export function JSXEmptyExpression() {}
+
+export function JSXFragment(node: Object) {
```

Comment:
should have a test for printing too

Reply:
Added printing test in this commit: https://github.com/babel/babel/pull/6552/commits/eef5e94b6a93452412b588c433a86cdd4d961f17

Wondering why the test folders are prefixed with XJS instead of JSX. 🤔

Reply:
funny, 😄 that used to be a thing, can ask the team - we should rename in another pr